### PR TITLE
Added animate feature and fixed tiling

### PR DESCRIPTION
**Pull request by fernandolis10**

### CODE REVIEW

File: GIBSDownloader/gibs_downloader.py [hunk 1]
```diff
@@ -5,44 +5,60 @@
...
                 print('Downloading:', date)
-            TiffDownloader.download_area_tiff(region, date.strftime("%Y-%m-%d"), download_path, xml_path, originals_path, product)
+            TiffDownloader.download_area_tiff(region, date.strftime("%Y-%m-%d"), xml_path, originals_path, product)
     else:
         print("The specified region and set of dates has already been downloaded")
 
-def tile_originals(tile_res_path, originals_path, tile, logging):
+def tile_originals(tile_res_path, originals_path, tile, logging, region):
     if not os.path.isdir(tile_res_path):
             os.mkdir(tile_res_path)
```
Comment (meherkasam) [inline GIBSDownloader/gibs_downloader.py:39]: Is this double-indented?

File: GIBSDownloader/gibs_downloader.py [hunk 2]
```diff
@@ -66,6 +82,16 @@ def remove_originals(originals_path, logging):
     shutil.rmtree(originals_path)
     os.mkdir(originals_path)
 
+def generate_video(originals_path, region, dates, video_path, xml_path, product):
+    if not os.path.isdir(video_path):
+        print("Generating video...", end="")
+        os.mkdir(video_path)
+        Animator.format_images(originals_path, region, dates, video_path, xml_path, product)
+        Animator.create_video(video_path)
+        print("done!")
```
Comment (meherkasam) [inline GIBSDownloader/gibs_downloader.py:91]: Might want to be more descriptive here with the output such as `Video generation has finished!`.

**Review by meherkasam — APPROVED** [review]

File: GIBSDownloader/tiff_downloader.py [hunk 3]
```diff
@@ -5,24 +5,25 @@
...
+
     @classmethod
-    def download_area_tiff(cls, region, date, xml_path, output, product, width=None, height=None, out_frmt="GTiff"):
-        """
-        region: rectangular region to be downloaded
-        date: YYYY-MM-DD
-        output: path/to/filename (do not specify extension)
-        returns tuple with dowloaded width and height
-        """
+    def download_area_tiff(cls, region, date, xml_path, filename, product, width=None, height=None, out_frmt="GTiff"):
```
Comment (meherkasam) [inline GIBSDownloader/tiff_downloader.py:14]: You removed the doc here. Was that intentional?

Comment (fernandolis10) [reply GIBSDownloader/tiff_downloader.py:14]: Yeah, those were old docs that no longer matched the function and I wanted to keep consistent with the rest of the package

**Pull request merged**